content/modules/redisgears/redisgears-installation.md

```diff
@@ -0,0 +1,204 @@
+---
```

Comment:
@bmansheim This entire file needs to be left out for now, we are still defining this instructions

Reply:
OK. I'll set it to draft.